blocks/library/cover-image/index.js

```diff
@@ -0,0 +1,169 @@
+/**
+ * WordPress dependencies
+ */
+import Placeholder from 'components/placeholder';
```

Comment:
We should use `import { Placeholder } from 'components';` to avoid duplication in the built files. (We should probably make this clearer in our dev docs)